src/futils.cpp

```diff
@@ -54,13 +55,15 @@ extern int strerror_r(int errnum, char *buf, size_t n);
 #endif
 
 namespace Exiv2 {
-    const char* ENVARDEF[] = {"/exiv2.php", "40"}; //!< @brief default URL for http exiv2 handler and time-out
-    const char* ENVARKEY[] = {"EXIV2_HTTP_POST", "EXIV2_TIMEOUT"}; //!< @brief request keys for http exiv2 handler and time-out
+    std::vector<std::string> ENVARDEF = {"/exiv2.php", "40"};
```

Comment:
Ahggg damn it! with c++98 we cannot do basic stuff like this ... Honestly, I would prefer to stay with the C-style arrays and throw the exception manually rather than add all the boiler plate needed to initialise a std::vector in C++98. 

Note: this compiled in Visual Studio 2017 since it is using all the power of modern c++ by default. There is no way to force it to use C++98. Recently they added some new flags to chose which standard to use ... but it starts from c++14:

https://blogs.msdn.microsoft.com/vcblog/2016/06/07/standards-version-switches-in-the-compiler/

In gcc and clang, this failed.

Reply:
I will drop the latest commit and throw `std::out_of_range` manually. Whenever we change to modern c++, we could replace that piece of code easily.

Reply:
What the hell, you can't tell visual studio to use C++98? That sounds like a pretty crazy decision.